package.json

```diff
@@ -62,7 +62,7 @@
     "start": "node scripts/buildtools start",
     "start:test:app": "yarn workspace @okta/test.app start",
     "mock:device-authenticator": "dyson playground/mocks/spec-device-authenticator/ 6512",
-    "clean:types": "rimraf ./types/generated",
+    "clean:types": "grunt clean:types",
```

Comment:
why this is better?

Reply:
We don't want to wipe the whole folder, there is a README in there, so I am using a glob to only delete declaration files